cmd/kev/cmd/dev.go

```diff
@@ -167,9 +167,21 @@ func runDevCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// initial manifests generation for specified environments only
+	if err := runRenderCmd(cmd, args); err != nil {
+		return err
+	}
+
 	if skaffold {
-		profileName := kevenv + kev.EnvProfileNameSuffix
-		go kev.RunSkaffoldDev(cmd.Context(), cmd.OutOrStdout(), []string{profileName}, namespace, kubecontext, 1000)
+		if skaffoldConfigPath, skaffoldConfig, ok := kev.ActivateSkaffoldDevLoop(workDir); ok {
+			if err := writeTo(skaffoldConfigPath, skaffoldConfig); err != nil {
+				log.Warnf("Couldn't write Skaffold config: %s.", err)
+				return err
```

Comment:
How about if we simply handle this as a log.Error (and use `return displayError(err)`)?

Reply:
updated